packages/flutter/lib/src/material/drawer.dart

```diff
@@ -194,10 +204,12 @@ class DrawerControllerState extends State<DrawerController> {
     }
   }
 
+  /// Starts an animation to open the drawer.
```

Comment:
is this the expected entry-point for programmatically opening drawers? Or is there some method on Scaffold or something to do it?